library/person.rb

```diff
@@ -1,4 +1,5 @@
 require './corrector'
+require './rentals'
 
 class Person
   def initialize(age:, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] Getters and Setters for @rentals should be present in this class. Let's add them
  `attr_accessor :name, :age, :rentals`